examples/tensorize_vllm_model.py

```diff
@@ -23,46 +23,63 @@
 # yapf: disable
 """
 tensorize_vllm_model.py is a script that can be used to serialize and 
-deserialize vLLM models. These models can be loaded using tensorizer directly 
-to the GPU extremely quickly. Tensor encryption and decryption is also 
-supported, although libsodium must be installed to use it. Install
-vllm with tensorizer support using `pip install vllm[tensorizer]`.
+deserialize vLLM models. These models can be loaded using tensorizer 
+to the GPU extremely quickly over an HTTP/HTTPS endpoint, an S3 endpoint,
+or locally. Tensor encryption and decryption is also supported, although 
+libsodium must be installed to use it. Install vllm with tensorizer support 
+using `pip install vllm[tensorizer]`.
 
-To serialize a model, you can run something like this:
+To serialize a model, you can run something like this from the root directory:
 
-python tensorize_vllm_model.py \
+python -m examples.tensorize_vllm_model \
```

Comment:
Did you actually get this to work without installing `vllm` from source? I was not able to because of the missing custom ops.

Reply:
I believe the env I was using was installing from source. I'll make a note of this.